src/pages/components/drawer.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Drawer
+description: >
+  A Drawer is a container for supplementary content that is anchored to the edge of a page.
+package: '@zendeskgarden/react-modals'
+components:
+  - modals/src/elements/DrawerModal/DrawerModal.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>
+        To display information or actions that are supplementary to the screen’s primary content
+      </li>
+      <li>To display a list of actions that affect the screen’s content, such as filtering data</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To obscure the main content and focus the user on modal content, use a
+          [Modal](/components/modal)
+        </Markdown>
+      </li>
+      <li>
+        <Markdown>
+          To display a small amount of content that contains actions, use a Tooltip Modal
+        </Markdown>
```

Comment:
This sentence doesn't flow well, I had to re-read a few times to understand. Can you reword it?

Reply:
I will omit this for now as well. I think this reaches a little too far for the utility of this section. The big benefit of `When not to use` section was to help make a decision when applications are close and I don't think that is a likely confusion here since Tooltips are for contextual use on the page.